src/main/kotlin/org/rust/lang/core/psi/impl/RustStubbedItemImpl.kt

```diff
@@ -0,0 +1,26 @@
+package org.rust.lang.core.psi.impl
+
+import com.intellij.lang.ASTNode
+import com.intellij.psi.stubs.IStubElementType
+import com.intellij.psi.util.PsiTreeUtil
+import org.rust.lang.core.psi.RustItem
+import org.rust.lang.core.psi.RustNamedElement
+import org.rust.lang.core.psi.RustOuterAttr
+import org.rust.lang.core.psi.RustVis
+import org.rust.lang.core.stubs.RustItemStub
+
+open class RustStubbedItemImpl : RustStubbedNamedElementImpl<RustItemStub>, RustItem {
+
+    constructor(node: ASTNode) : super(node)
+
+    constructor(stub: RustItemStub, nodeType: IStubElementType<*, *>) : super(stub, nodeType)
+
+    override fun getOuterAttrList(): List<RustOuterAttr> =
+        PsiTreeUtil.getChildrenOfTypeAsList(this, RustOuterAttr::class.java)
```

Comment:
Beware, that this code would trigger re-parse for the containing file since outer-attrs aren't part of the stub-tree.